### PR TITLE
The updated pyramid-jinja2 scaffold already sets up test requirements …

### PR DESCRIPTION
**Pull request by dodumosu**
…in `setup.py`

### CODE REVIEW

File: docs/quick_tour.rst [hunk 1]
```diff
@@ -678,10 +678,10 @@ egregious, as Pyramid has had a deep commitment to full test coverage since
 before its release.
 
 Our ``pyramid_jinja2_starter`` scaffold generated a ``tests.py`` module with
-one unit test in it. To run it, let's install the handy ``pytest`` test runner
-by editing ``setup.py``. While we're at it, we'll throw in the ``pytest-cov``
-tool which yells at us for code that isn't tested. Insert and edit the
-following lines as shown:
+one unit test in it. It also setup ``setup.py`` with test requirements:
```
Comment (mmerickel) [inline docs/quick_tour.rst]: How about `configured setup.py` instead of `setup setup.py`?

Comment (dodumosu) [inline docs/quick_tour.rst]: works for me.

**Comment by mmerickel** [comment]
Why'd you close this?

**Comment by dodumosu** [comment]
sorry, i wasn't sure how to merge the changes you requested for this one, so i closed this and created #2820

**Comment by mmerickel** [comment]
It's better to keep all of the work and review in one spot. I'd prefer you close the other PR and re-open this one.

As far as handling the changes you can either add another commit like you did or you can amend the previous commit and force-push.

Finally, this PR is against 1.7-branch which is fine but we also need to update master. Would you like me to forward-port the changes to master, or will you submit another PR to do that?

**Comment by dodumosu** [comment]
I've closed that and will be reopening this. Please forward-port the changes, thank you.

**Comment by mmerickel** [comment]
Thanks @dodumosu!

**Pull request merged**